Documentation/Tools/HolographicRemoting.md

```diff
@@ -0,0 +1,60 @@
+# Getting Started with Holographic Remoting
+
+Holographic remoting streams holographic content from a PC to your Microsoft HoloLens in real-time, using a Wi-Fi connection. This feature can significantly increase developer productivity when developing mixed reality applications.
+
+## Initial setup
+
+To enable remoting to a HoloLens, it is important to ensure that the project is using the latest remoting components.
+
+1. Open **Window > Package Manager**
+    - Verify that latest version of the **Windows Mixed Reality** package is installed.
+1. Ensure the latest Holographic Remoting application is installed, on the HoloLens, via the Microsoft Store.
+
+When using a HoloLens 2, support for remoting articulated hand and eye tracking data has been added to MRTK. To enable these features,
+configure the project using the following steps.
+
+1. Run the MRTK Configurator Utility (**Mixed Reality Toolkit > Utilities > Configure Unity Project**)
+2. Expand **Modify Configurations**
```

Comment:
Nit: Be consistent with using 1.2.3.4 or 1.1.1.1 (1.1.1.1 is used everywhere else here)

Reply:
oops. this was a copy paste error from a different version of this doc.

Reply:
fixed